.github/workflows/ci.yml

```diff
@@ -213,6 +213,35 @@ jobs:
       PLAYWRIGHT_BROWSER: ${{ matrix.browser }}
       PLAYWRIGHT_TRACES_DIR: playwright_traces
       PLAYWRIGHT_TRACE: 'true'
+      KAFKA_SERVERS: localhost:9092
+
+    services:
+      broker:
+        image: bitnamilegacy/kafka:3.5.1
+        env:
```

Comment:
We don't want a broker service here, we want it to be part of the compose stack. You should add the broker override to compose file:
- https://github.com/ONSdigital/dis-wagtail/blob/chore/use-kafka-backend-in-tests-dev-CMS-757/functional_tests/docker-compose.yml
- https://github.com/ONSdigital/dis-wagtail/blob/chore/use-kafka-backend-in-tests-dev-CMS-757/functional_tests/docker-compose-dev.yml

Reply:
This came to my mind of adding it to a compose file, then I somehow realised we actually have dedicated Docker-Compose files for `functional_tests`. Mind was blurry. Will get on it and update the comment above.

Reply:
I am such an idiot sandwich. Reading the makefile `functional-tests-up` and it being used in the CI step ` run: make functional-tests` should of gave this away.

Reply:
[10cace9](https://github.com/ONSdigital/dis-wagtail/pull/379/commits/10cace9b8d67c3ab9afea4fb6c198301b3285b55)